Controllers/authHandler.js

```diff
@@ -0,0 +1,184 @@
+const Customer = require("../models/customer");
+const bcrypt = require("bcrypt");
+var jwt = require("jsonwebtoken");
+const Session = require("../models/Session");
+const {
+  signupSchema,
+  loginSchema,
+  emailSchema,
+  passwordSchema,
+} = require("../validation/auth");
+
+const jwtSecret = "ghdsjvbiuy43870yoh22#$hoihnAkpvbjhTN)3&_N";

```

Comment:
Please remove this secret from this file as it can cause security leak. Consider using a .env file.